flow/api/main.go

```diff
@@ -11,26 +11,29 @@ import (
 	"flow/api/calendar"
 	"flow/api/data"
 	"flow/api/env"
+	"flow/api/middleware"
 	"flow/api/parse"
 	"flow/api/serde"
 
 	"flow/common/db"
 
 	"github.com/go-chi/chi/v5"
-	"github.com/go-chi/chi/v5/middleware"
+	chi_middleware "github.com/go-chi/chi/v5/middleware"
 )
 
+
 func setupRouter(conn *db.Conn) *chi.Mux {
 	router := chi.NewRouter()
 
 	router.Use(
 		// Responses are typically JSON, with the notable exception of webcal.
 		// We set the most common type here and override it as necessary.
-		middleware.SetHeader("Content-Type", "application/json"),
-		middleware.Logger,
-		middleware.Recoverer,
-		middleware.RequestID,
-		middleware.Timeout(10*time.Second),
+		middleware.CorsMiddleware(), // Enable CORS for development environments
```

Comment:
Hmm I think we want to enable this _only_ if we're running the Docker container locally - we don't want this to affect prod such that any requests coming from localhost bypass CORS

Maybe we could read in an env var for local dev and only do a `Router.Use` for this middleware if a dev flag is set